data/transactions/logic/eval.go

```diff
@@ -1699,6 +1699,36 @@ func opDig(cx *evalContext) {
 	cx.stack = append(cx.stack, sv)
 }
 
+func opCover(cx *evalContext) {
+	depth := int(cx.program[cx.pc+1])
+	idx := len(cx.stack) - 1 - depth
+	// Need to check stack size explicitly here because checkArgs() doesn't understand cover
+	// so we can't expect our stack to be prechecked.
```

Comment:
This may not be true any more with the changes to checkStack.  I think that now opCover _is_ sure to have a big enough stack to work with.

Reply:
good point, i'll re visit this